### DISCUSSION TITLE
Log mode format (for creating chronological content)

### DISCUSSION THREAD

**Discussion by ReessKennedy** [outline/outline#4136] [Ideas]
Outline is awesome but the content hierarchies make it a little difficult if you want to also create something close to a blog which would allow you to summarize your work completed for the day in a chronological format. Of course, you could created groups for year and month and work to keep it organized but there are many times I would like to just bullet point my accomplishments for the day and share them with the team and easily link these little chronological summaries to pages within Outline. Yes, I could do this in slack but putting it in Outline makes it more official and clear. It doesn't get lost this way. It would, optimally, be great for team members to be able to see a filterable list of date-based pages that could be filtered by author and date to become updated on happenings within the org. 

I know people will probably say, "out of scope" and "we need to focus on one thing" as this would start to resemble a blog-like feature but I think it could be a way to differentiate Outline from some others without some journaling feature. This idea also coincides with a hug upswing in the usage of products like Roam and Obsidian and LogSeq which are based on a PAGES + JOURNAL architecture. 

For consideration! And thanks for all the great work from the devs and this community.

**Comment by tommoor** [comment]
I don’t think it’s entirely out of scope, do you have any more thoughts
about how a log mode would work? How would it differ in your eyes from what
we have today?

**Reply by ReessKennedy** [reply]
In general, I just think Outline has this great team inclusion functionality but then that team has to go elsewhere to write up posts on "what they are working on now" and this context-switching is really unfortunate and where other solutions are doing a better job at accommodating more dynamic team knowledge sharing (will include examples after). I realize that Outline is coupled with Slack and that this could be a place for blog-like updates from team members and discussion but Slack isn't open source and slack has so much noise in it that actual thoughtful progress reports get lost in the ephemera. As a leader, I want be able to write a weekly summary of what I worked on and include links to places inside the docs in this summary blog post that I either updated or think need attention and then also allow for team members to comment on this post and include their own notes or author their own posts and allow us, in outline, to filter these posts by team member like on a Wordpress blog.

Examples of blog + permanent notes + team collab: Athens Research (They are working on a collaborative feature)
Examples of blog + permanent notes: Roam, Obsidian, LogSeq (these all follow this format that we have these daily notes and then we have the evergreen stuff that we built wikis and pages for)
Example of other note taking apps that have calendar views: Ample notes and many more
Example of a general-purpose wiki that allows you to also build in your own blog-like database of updates: Notion
Example of open source team blog: Automattic's P2 has gotten a facelift and this might be a better example of something that could inspire Outline ... https://wordpress.com/p2/ ... it looks like Wordpress didn't release the theme for this to the open source community and is charge $8/u/m for teams to use this. The older version of the theme is open source and in the theme directory.

**Reply by ReessKennedy** [reply]
I've tried to make something like this happen in Outline but it's just a hack and isn't very good/elegant and gets messy with other team members and there isn't any commenting. 
<img width="861" alt="Screen Shot 2022-11-06 at 8 53 19 PM" src="https://user-images.githubusercontent.com/3970267/200210915-13b44650-64f0-4c94-aefe-5abdc49a1c00.png">

**Reply by tommoor** [reply]
Can you help me understand what is preventing daily notes being individual docs in the same chronological collection?
